server/Models/SurrenderApplication.js

```diff
@@ -0,0 +1,50 @@
+import pg from "pg"
+
+const pool = new pg.Pool({
+    connectionString: "postgres://postgres:password@localhost:5432/pets-app"
+})
+
+class SurrenderApplication{
+  constructor({id = null, name, phoneNumber, phone_number, email, petName, petAge,    petType, petImage, vaccinationStatus = "No"}){
+    //surrender_application fields
+    this.id = id;
+    this.name = name;
+    this.phoneNumber = phone_number || phoneNumber;
+    this.email = email;
+    this.status = "Pending";
+    //This needs to be set at record insert time, after the adoptable_pet is created
+    this.adoptable_pet_id = null;
+    //adoptable_pets fields
+    this.petName = petName;
```

Comment:
The surrender application only needs:
```
name
phone number
email
adoptable_pet_id
status (default "pending")
```
to normalize our data, we create an association between the surrender form and creating an adoptable pet. Otherwise we have to keep track of this adoptable pet data in 2 places - both the surrender form and on the pet